i18n/fi.po

```diff
@@ -0,0 +1,1222 @@
+#: templates/features.html:169 templates/features.html:183
+#: templates/features.html:256 templates/features.html:277
+#: templates/features.html:299 templates/features.html:318
+#: templates/features.html:331
+msgid "Documentation"
+msgstr "Dokumentaatio"
+
+#: templates/base.html:34 templates/support.html:4
+msgid "Support"
+msgstr "Tuki"
+
+#: templates/base.html:35 templates/development.html:4
+#: templates/development.html:8
+msgid "Development"
+msgstr "Kehitys"
+
+#: templates/base.html:36 templates/donate.html:4
+msgid "Donate"
+msgstr "Lahjoita"
+
+#: templates/base.html:37
+msgid "Online Shell"
+msgstr "Online-komentokehoite"
+
+#: templates/base.html:51
+msgid " Compute with Gamma "
+msgstr "Laske Gamman avulla"
+
+#: templates/base.html:54
+msgid "Compute"
+msgstr "Laske"
+
+#: templates/base.html:58
+msgid "Download Now"
+msgstr "Lataa nyt"
+
+#: templates/base.html:60
+msgid "Latest Version"
+msgstr "Uusin versio"
+
+#: templates/base.html:61
+msgid "Development Version"
+msgstr "Kehitysversio"
+
+#: templates/base.html:65
+msgid "Quick Links"
+msgstr "Pikalinkit"
+
+#: templates/base.html:68
+msgid "Downloads (source tarballs)"
+msgstr "Lataukset (lähdekoodin tarball-tiedostot)"
+
+#: templates/base.html:69
+msgid "Mailing list"
+msgstr "Postituslista"
+
+#: templates/base.html:70
+msgid "Source code"
+msgstr "Lähdekoodi"
+
+#: templates/base.html:71
+msgid "Issues tracker"
+msgstr "Bugiseuranta"
+
+#: templates/base.html:72
+msgid "Wiki"
+msgstr "Wiki"
+
+#: templates/base.html:73
+msgid "Try SymPy online now"
+msgstr "Kokeile SymPyn online-versiota"
+
+#: templates/base.html:74
+msgid "Official SymPy blog"
+msgstr "Virallinen SymPy-blogi"
+
+#: templates/base.html:75
+msgid "Planet SymPy"
+msgstr "Planet SymPy - blogi"
+
+#: templates/base.html:76
+msgid "Chat (Gitter)"
+msgstr "Gitter-Chat"
+
+#: templates/base.html:77
+msgid "IRC channel logs"
+msgstr "IRC-kanavan lokitiedostot"
+
+#: templates/base.html:81
+msgid "Follow Us"
+msgstr "Seuraa meitä"
+
+#: templates/base.html:112
+#, python-format
+msgid ""
+"This page is open source. Fork <a href=\"%(url)s\">the project on GitHub</a> "
+"to edit it."
+msgstr ""
+"Tämä sivu on avointa lähdekoodia. Luo haara <a href=\"%(url)s\">projektista "
+"GitHubissa</a> muokataksesi sitä."
+
+#: templates/base.html:115
+msgid "Languages"
+msgstr "Kielet"
+
+#: templates/development.html:11
+msgid ""
+"We are interested in any contribution. There are many ways how you can\n"
+"    contribute"
+msgstr ""
+"Me olemme kiinnostuneita mistä tahansa avustuksesta. On olemassa monta tapaa "
+"jolla voit\n"
+"      avustaa"
+
+#: templates/development.html:15
+msgid ""
+"If you wrote anything interesting using SymPy, please send us a patch with "
+"the\n"
+"        code (see below for more information)"
+msgstr ""
+"Mikäli kirjoitit jotakin kiinnostavaa SymPy-ohjelmistolla, ole hyvä ja "
+"lähetä meille korjaustiedost\n"
+"         lähdekoodilla (ole hyvä ja katso lisätiedot alla)"
```

Comment:
korjaustiedosto (missing last letter)

Translating "with" to `-lla` is probably not 100% formal language, could be rephrased.